common/engine_legacy_conf/service.hh

```diff
@@ -190,7 +190,7 @@ class service : public object {
   std::string _host_name;
   std::string _icon_image;
   std::string _icon_image_alt;
-  uint32_t _initial_state;
+  opt<unsigned int> _initial_state;
```

Comment:
are you sure we have to make this field inheritable?

Reply:
will fix, the fields will be non-configurable.